### PR TITLE
Added aryasoni98

### PR DESCRIPTION
**Pull request by aryasoni98**
- Added Data ( aryasoni.json )
- Rename Tab Title
- Update package-lock, package page 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/86"></a>

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,10 +24,10 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Link Free</title>
   </head>
   <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <noscript>Open source alternative to LinkTree</noscript>
```
Comment (eddiejaoude) [inline public/index.html:30]: Thank you for these changes 👍

File: public/data/aryasoni.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      "url": "http://github.com/aryasoni98",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "http://twitter.com/aryasoni98",
+      "icon": "twitter"
+    }
+  ]
+}
```
Comment (eddiejaoude) [inline public/data/aryasoni.json:17]: This should be in a separate PR to other app changes as there are unrelated and both add different value

File: package.json [hunk 3]
```diff
@@ -13,7 +13,7 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-router-dom": "^5.3.0",
-    "react-scripts": "4.0.3",
+    "react-scripts": "^1.1.5",
```
Comment (eddiejaoude) [inline package.json:16]: Not sure why this has changed?

File: public/index.html (hunk 1 above)
Comment (aryasoni98) [reply public/index.html:30]: Thank You 😊

File: public/data/aryasoni.json (hunk 2 above)
Comment (aryasoni98) [reply public/data/aryasoni.json:17]: Okay 👍 I will make it

File: package.json (hunk 3 above)
Comment (aryasoni98) [reply package.json:16]: By mistake, I just push

**Comment by aryasoni98** [comment]
So should I close the PR and Resend @eddiejaoude

**Comment by eddiejaoude** [comment]
@aryasoni98 it is up to you, whatever is easier. You can either update this PR or close and create a new one, either way is fine with me 🤓

**Comment by nhcarrigan** [comment]
@aryasoni98 can you give this a rebase when you have a moment? If you have any questions, let us know.
